test/parallel/test-trace-events-environment.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
```

Comment:
Please describe the test, as per https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md

Reply:
Done